ECMA2Yaml/ECMAHelper/YamlGenerators/SDPTOCGenerator.cs

```diff
@@ -91,9 +95,32 @@ private static TocViewModel GenerateTocItemsForMembers(Models.Type t)
                     Name = m.DisplayName
                 };
                 tocEntry.Metadata["type"] = m.ItemType.ToString();
+                if (IsNeedAddMonikers(t.Monikers, m.Monikers))
+                {
+                    tocEntry.Metadata[OPSMetadata.Monikers] = m.Monikers.ToArray();
+                }
                 items.Add(tocEntry);
             }
             return items;
         }
+
+        private static bool IsNeedAddMonikers(HashSet<string> tMonikers, HashSet<string> mMonikers)
+        {
+            if (mMonikers?.Count > 0)
```

Comment:
it can be as simple as `return mMonikers?.Count > 0 && tMonikers?.Count > 0 && !tMonikers.SetEquals(mMonikers);`